Packs/Slack/Integrations/SlackV3/SlackV3.py

```diff
@@ -1209,115 +1408,70 @@ async def listen(client: SocketModeClient, req: SocketModeRequest):
     try:
         data: dict = req.payload
         event: dict = data.get('event', {})
-        subtype = data.get('subtype', '')
         text = event.get('text', '')
-        user_id = event.get('user', '')
+        user_id = data.get('user', {}).get('id', '')
+        if not user_id:
+            user_id = event.get('user', '')
         channel = event.get('channel', '')
-        message_bot_id = data.get('bot_id', '')
         thread = event.get('thread_ts', None)
         message = data.get('message', {})
-        entitlement_reply = None
-        action_text = None
+        action_text = ''
         message_ts = message.get('ts', '')
+        actions = data.get('actions', [])
+
         # Check if slash command received. If so, ignore for now.
         if data.get('command', None):
             demisto.debug("Slash command event received. Ignoring.")
             return
 
-        actions = data.get('actions', [])
-        integration_context = get_integration_context(SYNC_CONTEXT)
-        if subtype == 'bot_message' or message_bot_id or message.get('subtype') == 'bot_message' \
-                or \
-                event.get('bot_id', None):
-
-            if integration_context.get('bot_user_id'):
-                bot_id = integration_context['bot_user_id']
-                if bot_id == 'null' or bot_id is None:
-                    # In some cases the bot_id can be stored as a string 'null', this handles this edge case.
-                    bot_id = await get_bot_id_async()
-                    set_to_integration_context_with_retries({'bot_user_id': bot_id}, OBJECTS_TO_KEYS, SYNC_CONTEXT)
-            else:
-                bot_id = await get_bot_id_async()
-                set_to_integration_context_with_retries({'bot_user_id': bot_id}, OBJECTS_TO_KEYS, SYNC_CONTEXT)
-            if event.get('subtype') == 'bot_message':
-                demisto.debug("Received bot_message event type. Ignoring.")
-                return
-            if event.get('bot_id', '') == bot_id:
-                demisto.debug("Received bot message from the current bot. Ignoring.")
-                return
+        # Check to see if the event is about a newly handled event.
+        elif event.get('type') == 'channel_created':
```

Comment:
handle this only if mirroring is active.

Reply:
I don't think we need to. Mirroring is only active when long running is active. We don't have a parameter to isolate only mirroring. We could add one though. WDYT?